lib/medtronic/processData.js

```diff
@@ -27,7 +27,8 @@ var savedFileEntry, fileDisplayPath;
 var debug = (typeof __DEBUG__ === 'undefined') ? false : __DEBUG__;
 var isBrowser = typeof window !== 'undefined';
 
-var larger = true; //TODO: this should be based on model number
+var larger = false; // model 523 has larger records
```

Comment:
523 has larger records than which other model(s)? Just for a bit more clarity here.